native/src/modules/app/components/NavigationTypes.js

```diff
@@ -5,59 +5,59 @@ import type { RouteProp } from '@react-navigation/native'
 import type { FeedbackInformationType } from '../../../routes/feedback/containers/FeedbackModalContainer'
 import { LanguageModel, OfferModel } from 'api-client'
 
-export type IntroRouteType = 'Intro'
```

Comment:
minor: From my react-navigation experience it is common to use uppercase route names

Reply:
Hmmm. I changed to lowercase to have a direct mapping to urls.